cg/store/api/find.py

```diff
@@ -172,6 +172,10 @@ def panel(self, abbrev):
         """Find a panel by abbreviation."""
         return self.Panel.query.filter_by(abbrev=abbrev).first()
 
+    def panels(self):
+        """Returns all panels."""
+        return self.Panel.query.order_by(models.Panel.abbrev)
```

Comment:
I guess the panels are actually really stored in scout, right? Do you know when we are creating the panels in statusdb?

Reply:
I think we are adding them one-by-one